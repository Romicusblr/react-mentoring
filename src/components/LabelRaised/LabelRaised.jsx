import React from 'react';
import PropTypes from 'prop-types';
import classNames from 'classnames';
import style from './LabelRaised.module.css';

function LabelRaised({ children, className, ...other }) {
  return (
    <span
      className={classNames(style.labelRaised, className)}
      {...other}
    >
      {children}
    </span>
  );
}

LabelRaised.propTypes = {
  children: PropTypes.node.isRequired,
  className: PropTypes.string,
};

LabelRaised.defaultProps = {
  className: '',
};

export default LabelRaised;
