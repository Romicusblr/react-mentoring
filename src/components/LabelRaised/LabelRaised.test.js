import LabelRaised from '.';

fdescribe('LabelRaised', () => {
  it('should be selectable by class "labelRaised"', () => {
    expect(shallow(<LabelRaised />).is('.labelRaised')).toBe(true);
  });

  it('should match to snapshot', () => {
    const snap = shallow(<LabelRaised />);
    expect(snap).toMatchSnapshot();
  });
});
