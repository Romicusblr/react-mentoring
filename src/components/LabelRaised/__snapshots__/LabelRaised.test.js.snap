// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LabelRaised should match to snapshot 1`] = `
<span
  className="labelRaised"
/>
`;
